uitools/import/Esri/ArcGISRuntime/Toolkit/BasemapGallery.qml

```diff
@@ -241,6 +246,34 @@ Pane {
                 anchors.fill: parent
                 hoverEnabled: true
                 onClicked: controller.setCurrentBasemap(modelData.basemap)
+
+                // When mouse enters thumbnail area, use timer to delay showing of tooltip.
+                onEntered: {
+                    timerOnEntered.start();
+                }
+                // When mouse exits thumbnail area, use timer to delay hiding of tooltip.
+                onExited: {
+                    timerOnExited.start();
+                }
+
+                Timer {
+                    id: timerOnEntered
+                    interval: 1000
+                    repeat: false
+                    onTriggered: {
+                        if (allowTooltips && mouseArea.containsMouse && modelData.tooltip !== "")
+                            basemapDelegate.ToolTip.visible = true;
+                    }
+                }
+
+                Timer {
+                    id: timerOnExited
+                    interval: 1000
```

Comment:
:thought_balloon: What's the look and feel of this? There were complaints previously that the tooltip appeared to quickly, and 1 second feels relatively short. it might be worth comparing to:

```qml
Qt.styleHints.mousePressAndHoldInterval * 2
```

Reply:
The `Qt.styleHints.mousePressAndHoldInterval` is 800ms.

To be consistent with the comments received previously, the interval of the two timers have been changed to `Qt.styleHints.mousePressAndHoldInterval * 2`.